formulaic/templates/admin/formulaic/form/change_form.html

```diff
@@ -14,7 +14,7 @@
 
   <meta name="ember-formulaic/config/environment" content="{{ environment_config }}" />
 
-      <script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/tinymce.min.js" referrerpolicy="origin"></script>
```

Comment:
Can you make this configurable in the project settings? First, what happens if you use `no-api-key`? Does it still work?  If so, make that the default, and allow it to be overridden with a new setting: `FORMULAIC_TINYMCE_API_KEY`. Then set that value in the other project.